scripts/generate-routes.ts

```diff
@@ -1,6 +1,6 @@
 import { writeFile } from "fs/promises";
 import { resolve } from "path";
-import chalk from "chalk";
+import { green, red } from "chalk";
```

Comment:
There was an issues about chalk import outside of default, please revert back them